### ISSUE TITLE
Unit with paused movement can still turn

### ISSUE THREAD

**Issue by MlemandPurrs** [OpenRA/OpenRA#20837]
A unit that have got it's movement paused with a condition can still turn in place when auto-acquiring targets in range.

With a setup like this, expect it to not turn while under effect of condition granted on attack, instead it keeps turning to face target.
```
FTNK:
    Inherits: ^TrackedVehicle
    Inherits@Crates: ^AcceptsAllCrates
    Inherits@AUTOTARGET: ^AutoTargetGroundAssaultMove
    Valued:
        Cost: 800
    Tooltip:
        Name: Flame Tank
    Buildable:
        BuildDuration: 1350
        BuildDurationModifier: 100
        BuildPaletteOrder: 7
        Prerequisites: fix
        Queue: Vehicle.Nod
        Description: Heavily armored flame-throwing vehicle.\n\nStrong vs Infantry, Buildings.\nWeak vs Tanks.
    Mobile:
        Locomotor: tracked-crusher
        TurnSpeed: 12
        Speed: 90
        PauseOnCondition: dont-move || emp-crate-collected || boarding
    Health:
        HP: 35000
    Armor:
        Type: Heavy
    RevealsShroud:
        Range: 4c128
    Armament:
        Weapon: BigFlamer
        LocalOffset: 512,128,42, 512,-128,42
        MuzzleSequence: muzzle
        PauseOnCondition: emp-crate-collected
    AttackFrontal:
        FacingTolerance: 0
        Voice: Attack
        PauseOnCondition: emp-crate-collected
    AttackMove:
        Voice: Attack
    WithMuzzleOverlay:
    Explodes:
        Weapon: FlametankExplode
        EmptyWeapon: FlametankExplode
    GrantConditionOnAttack:
        Condition: dont-move
        RevokeDelay: 16
        RevokeOnNewTarget: False
        RevokeAll: True
```

**Comment by PunkPun** [comment]
duplicate of #19751

**Comment by MlemandPurrs** [comment]
oh, sorry.
